ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -230,6 +246,37 @@ const ProfileMainScreenFC = () => {
       />
       <VSpacer size={8} />
       <ContentWrapper>
+        {appFeedbackEnabled && surveyUrl && (
+          <ListItemAction
+            label={feedbackLabel}
+            icon="message"
+            variant="primary"
+            testID="feedbackButton"
+            onPress={() => {
+              void openAuthenticationSession(surveyUrl, "");
+            }}
+            accessibilityLabel={feedbackLabel}
+          />
+        )}
+        <ListItemAction
+          label={reviewLabel}
+          icon="starEmpty"
+          variant="primary"
+          testID="reviewButton"
+          onPress={() =>
+            openWebUrl(
+              Platform.select({
+                ios: "https://apps.apple.com/app/id1501681835?action=write-review",
+                // On Android we don't have a direct link to the review page we need to link the app page
+                android:
+                  "https://play.google.com/store/apps/details?id=it.pagopa.io.app",
+                default: "https://io.italia.it/"
```

Comment:
Do we really need a `default` case? By the way, if we do, wouldn’t it be better to take it from `absolutePortalLinksSelector`? What do you think?

Reply:
You're right! We can avoid the default web url